layouts/partials/ecosystem/registry/entry.html

```diff
@@ -141,6 +141,7 @@ <h4 class="alert-heading">Deprecated</h4>
                     {{ end -}}
                     {{- .description | markdownify -}}
                 </div>
+
```

Comment:
remove that extra line and it should disappear from the PR

Reply:
okay